edgexfoundry/edgex-compose
# refactor: Rework env overrides for common config in place

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Pull all latest images
Run the following commands and verify services bootstrap successfully and are using the appropriate MessageBus
### Secure Mode
- `make run ds-virtual ds-rest asc-sample`
  - Verify logs have : `Connected to redis Message Bus @ redis://edgex-redis:6379 with AuthMode='usernamepassword'`
- `make clean`
- `make run ds-virtual ds-rest asc-sample mqtt-bus`
  - Verify logs have : `Connected to mqtt Message Bus @ tcp://edgex-mqtt-broker:1883 with AuthMode='usernamepassword'`
- `make clean`
### Non-Secure Mode
- `make run ds-virtual ds-rest asc-sample no-secty`
  - Verify logs have : `Connected to redis Message Bus @ redis://edgex-redis:6379 with AuthMode='usernamepassword'`
- `make clean`
- `make run ds-virtual ds-rest asc-sample mqtt-bus no-secty`
  - Verify logs have : `Connected to mqtt Message Bus @ tcp://edgex-mqtt-broker:1883 with AuthMode='none'`
- `make clean`
